packages/mobile/src/hooks/useNavigation.ts

```diff
@@ -48,7 +58,10 @@ export const useNavigation = <
   return useMemo(
     () => ({
       navigate: performNavigation(nativeNavigation.navigate),
-      push: performNavigation(nativeNavigation.push),
+      push:
+        'push' in nativeNavigation
+          ? performNavigation(nativeNavigation.push)
+          : () => {},
```

Comment:
Maybe we should log something here? Might be confusing to someone calling `push` and nothing happening

Reply:
great idea